### PR TITLE
Document the different assets that power kubeflow.org and their owners.

### PR DESCRIPTION
**Pull request by jlewi**

### CODE REVIEW

**Comment by jlewi** [comment]
/assign @animeshsingh 

@animeshsingh Could you take a look at this?

**Comment by jlewi** [comment]
ping @animeshsingh ?

File: how-to/kubeflow_assets.md [hunk 1]
```diff
@@ -0,0 +1,164 @@
...
+
+* Password is stored in GCP Secret Manager project `kubeflow-admin`
+* You will also need a recovery token which are also stored in secret manager
+
+  * These are onetime use so remove it when you use it
+  * When you login you need to select use recovery token.
+
+## kubeflow.org GSuite Domain
+
+Admins
```
Comment (animeshsingh) [inline how-to/kubeflow_assets.md:29]: Believe this is immutable? Is anyone collecting overall github statistics for the org?

File: how-to/kubeflow_assets.md [hunk 2]
```diff
@@ -0,0 +1,164 @@
...
+
+https://medium.com/kubeflow
+
+Editors
+
+* @chrisheecho_78982
+* @ankitbahuguna
+* @jlewi
+* @edd
+* @thealamkin
```
Comment (animeshsingh) [inline how-to/kubeflow_assets.md:75]:
```suggestion
* @thealamkin
* @animation2007
```

File: how-to/kubeflow_assets.md [hunk 3]
```diff
@@ -0,0 +1,164 @@
...
+Root account:
+
+username: kubeflow
+password: 
+
+ * This is stored in Google Secret Manager inside project kubeflow-admins
+
+The following twitter accounts are linked as teammebers
+
+* @aronchick
```
Comment (animeshsingh) [inline how-to/kubeflow_assets.md:111]:
```suggestion
* @animeshsingh 
* @aronchick
```

File: how-to/kubeflow_assets.md (hunk 1 above)
Comment (jlewi) [reply how-to/kubeflow_assets.md:29]: What do you mean by GitHub statistics? Policy doesn't allow us for granting non-googlers admin privileges. Also following the principle of least privilege we don't want to grow the number of root users/admins.
In accordance with that, we are trying to great policy groups (https://github.com/kubeflow/internal-acls) that have privileges needed to fulfill different roles (e.g. test/release admin, drive manager etc...) and then add folks as necessary to those groups.

Ideally everything is automated through GitOps so there is very little need to have root privileges.

File: how-to/kubeflow_assets.md (hunk 2 above)
Comment (jlewi) [reply how-to/kubeflow_assets.md:75]: I assume animation2007 is you? I don't have permission to add you as an editor; I think maybe Thea does. This doc is only intended to capture the current set of permissions. So I suggest we leave it as is and then you can follow up with Thea if you want to be added as an editor.

**Comment by jlewi** [comment]
@animeshsingh I added you to the twitter team. For the others lets follow up. This PR was mainly intended to capture the state of the world and let people know who to contact about various resources.

File: how-to/kubeflow_assets.md (hunk 2 above)
Comment (animeshsingh) [reply how-to/kubeflow_assets.md:75]: sure

**Pull request merged**